meson.build

```diff
@@ -67,3 +67,4 @@ add_project_arguments(cxx.get_supported_arguments(test_warning_args),
 executable('hello', 'examples/hello.cpp', 'examples/boilerplate.cpp', dependencies: spidermonkey)
 executable('cookbook', 'examples/cookbook.cpp', 'examples/boilerplate.cpp', dependencies: spidermonkey)
 executable('repl', 'examples/repl.cpp', 'examples/boilerplate.cpp', dependencies: [spidermonkey, readline])
+executable('tracing', 'examples/tracing.cpp', 'examples/boilerplate.cpp', dependencies: [spidermonkey, readline])
```

Comment:
This doesn't need to link to readline, `dependencies: spidermonkey` should be fine.

Reply:
Good catch!